engine/render/renderer/Renderer.cpp

```diff
@@ -14,6 +14,15 @@
 #include "platform/vulkan/utils/Types.h"
 #include "statics/Statics.h"
 
+// TODO look into why including via GLFW header + volk isn't defining this - JDM
```

Comment:
@Raelr do you have any thoughts on solving this issue here? The relationship between GLFW and Vulkan is positively spaghetti.